CHANGES.rst

```diff
@@ -6,7 +6,7 @@ There's a frood who really knows where his towel is.
 2.2.1 (unreleased)
 ^^^^^^^^^^^^^^^^^^
 
-- solve issue #861.
+- The 'drop' event and its 'target' method are used to obtain the element (column) and not generate ambiguity (fixes # 861). [Mubra]
```

Comment:
use:

(fixes \`#861 \<https://github.com/collective/collective.cover/issues/861>`_)

to link to issue.

see version 2.1b2 in:

https://raw.githubusercontent.com/collective/collective.cover/master/CHANGES.rst

Reply:
thanks for the support @idgserpro